### PR TITLE
route-card full-rating order

### PR DESCRIPTION
**Pull request by ginold**
related to #243

### CODE REVIEW

File: c2corg_ui/static/js/card.js [hunk 1]
```diff
@@ -216,7 +214,22 @@ app.CardController.prototype.getFullRatings = function() {
...
     }
   }
-  return ratings;
+  var orderedRatings = [ratings['global_rating'], ratings['rock_rating'], ratings['aid_rating'], ratings['ice_rating'],
+    ratings['mixed_rating'], ratings['via_ferrata_rating'], ratings['engagement_rating'], ratings['risk_rating'],
+    ratings['equipment_rating'], ratings['exposition_rock_rating'], ratings['ski_rating'], ratings['labande_rating'],
+    ratings['hiking_rating'], ratings['snowshoe_rating'], ratings['mtb_rating'], doc['hiking_mtb_exposition']];
+
+  // we want to get the name of the prop to show it as a tooltip
+  $.each(orderedRatings, function(i, rating) {
```
Comment (asaunier) [inline c2corg_ui/static/js/card.js]: This code looks a bit complicated and confusing, I think.
And what happens if 2 ratings share the same value (for example 'exposition_rock_rating' and 'hiking_mtb_exposition')?

Isn't it possible to do something like:
```
var ratingOrder = ['global_rating', 'rock_rating', ... ];
ratingOrder.forEach(function(name) {
  if (prop in ratings && ratings[prop]) {
    fullRatings[prop] = ratings[prop];
  }
});
```
or even to use the array ``sort`` function https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/sort ?

Comment (ginold) [inline c2corg_ui/static/js/card.js]: :+1:  for your solution in the middle !

Comment (ginold) [inline c2corg_ui/static/js/card.js]: updated with your suggestion.

**Comment by asaunier** [comment]
Travis says
> /home/travis/build/c2corg/v6_ui/c2corg_ui/static/js/card.js
  198:1  error  Trailing spaces not allowed  no-trailing-spaces

Please always run "make check" before pushing to github :P

**Comment by ginold** [comment]
sorry chef! can i merge after that?

**Comment by asaunier** [comment]
> can i merge after that?

:+1:

**Pull request merged**